hypothesis-python/tests/conjecture/test_data_tree.py

```diff
@@ -47,64 +47,64 @@ def accept(tf):
 def test_can_lookup_cached_examples():
     @runner_for(b"\0\0", b"\0\1")
     def runner(data):
-        data.draw_bits(8)
-        data.draw_bits(8)
+        data.draw_integer(0, 8)
+        data.draw_integer(0, 8)
 
```

Comment:
there's a lot of work ahead of me in inspecting and fixing these tests! Usually these tests are relying on `draw_bits` just to record a draw to the tree, and it's safe to replace with a `draw_integer` call with equivalent bounds. But I think there are tests where this isn't the case and the test will need to be fixed further.

Reply:
Sounds good - though note that `draw_bits(8)` -> draw_integer(0, 255)`!